src/components/ChatEntry.js

```diff
@@ -1,22 +1,45 @@
 import React from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp';
 
 const ChatEntry = (props) => {
-  return (
-    <div className="chat-entry local">
-      <h2 className="entry-name">Replace with name of sender</h2>
-      <section className="entry-bubble">
-        <p>Replace with body of ChatEntry</p>
-        <p className="entry-time">Replace with TimeStamp component</p>
-        <button className="like">🤍</button>
-      </section>
-    </div>
+  
+  const onLikeButtonClick = () => {
+    const updatedChat  = {
+      id: props.id,
+      sender: props.sender,
+      body: props.body, 
+      timeStamp: props.timeStamp,
+      liked: !props.isLiked,
+    };
+    props.onUpdate(updatedChat);
+  };
+
+  let buttonContent = props.isLiked ? '❤️' : '🤍'; 
```

Comment:
This can be a `const` variable since `buttonContent` shouldn't be modified directly.